src/Kdyby/Annotations/DI/AnnotationsExtension.php

```diff
@@ -81,7 +82,7 @@ public function getConfig(array $defaults = NULL, $expand = TRUE)
 			$config = Nette\DI\Config\Helpers::merge($config, ['ignore' => $globalConfig['doctrine']['ignoredAnnotations']]);
 		}
 
-		return $this->compiler->getContainerBuilder()->expand($config);
+		return Nette\DI\Helpers::expand($config, $this->getContainerBuilder()->parameters);
```

Comment:
Yes. I only rarely need parameters in my extensions but when I do I do it this way. For example https://github.com/Arachne/Twig/blob/master/src/DI/TwigExtension.php#L48.